modules/drivers/mongo/src/metabase/driver/mongo/execute.clj

```diff
@@ -146,8 +148,8 @@
    ^ClientSession session
    stages timeout-ms]
   (let [coll      (.getCollection db coll)
-        pipe      (m.util/into-array-list (m.conversion/to-db-object stages))
-        aggregate (.aggregate coll session pipe BasicDBObject)]
+        pipe      (java.util.ArrayList. ^java.util.Collection (mongo.conversion/to-document stages))
```

Comment:
I suggest  importing these classes to make this line shorter.

Reply:
Done.